shared/actions/push.js

```diff
@@ -184,13 +242,30 @@ function* deletePushTokenSaga(): Saga.SagaGenerator<any, any> {
   }
 }
 
+function* _mobileAppState(action: AppGen.MobileAppStatePayload) {
```

Comment:
should remove the _. i know our sagas are inconsistent in this regard but i think it's likely cleaner to just remove it. 1. all the functions are named this way in this file and 2. we likely want to do this globally. 
I think the _ naming convention is useful if you're exporting things and also keeping things just in the module but by definition all the sags should only ever export the main (last) saga and nothing else so there should be no ambiguity